### PR TITLE
Support Decidim 0.26.4

### PR DESCRIPTION
**Pull request by ferblape**
#### :tophat: What? Why?

This PR updates the supported version of Decidim to 0.26.4. It's a minor version so the support is guaranteed

I've also included a couple of details:

- instead of forcing Ruby version to 2.7.5 I think is more convenient to use the same Ruby version requirement that Decidim, which is `>= 2.7`
- INMHO `Gemfile.lock` and `.ruby-version` shouldn't be included in the repo

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ tmp/
 spec/decidim_dummy_app/
 .rspec-failures
 .byebug_history
+.ruby-version
+Gemfile.lock
```
Comment (tramuntanal) [inline .gitignore]: I'm sorry I prefer to have both files checked in the repository

File: .ruby-version [hunk 2]
```diff
@@ -1 +0,0 @@
-2.7.7
```
Comment (tramuntanal) [inline .ruby-version]: Undo this change please

File: Gemfile.lock [hunk 3]
```diff
@@ -1,752 +0,0 @@
-PATH
```
Comment (tramuntanal) [inline Gemfile.lock:1]: Undo this change please

**Review by tramuntanal — CHANGES_REQUESTED** [review]
Thanks for that PR.

We're already releasing support for Decidim v0.27.
Can you please point this PR against the stable branch for 0.26 please? `release/0.26-stable`

File: .gitignore (hunk 1 above)
Comment (ferblape) [reply .gitignore]: Done

File: .ruby-version (hunk 2 above)
Comment (ferblape) [reply .ruby-version]: Done

File: Gemfile.lock (hunk 3 above)
Comment (ferblape) [reply Gemfile.lock:1]: Done

**Review by tramuntanal — CHANGES_REQUESTED** [review]
Thanks for the PR again and for the changes :smile: 

Can you run `bundle update decidim` in order to update the `Gemfile.lock` please?

**Comment by ferblape** [comment]
Done!
